TwilioVerifySDK/TwilioSecurity/Sources/Storage/AuthenticatedSecureStorage.swift

```diff
@@ -95,7 +95,11 @@ extension AuthenticatedSecureStorage: AuthenticatedSecureStorageProvider {
         success(data)
       } catch {
         Logger.shared.log(withLevel: .error, message: error.localizedDescription)
-        failure(error)
+        if let biometricError = BiometricError.given(error as NSError) {
```

Comment:
Maybe you could save some lines by using a default for this operation, my suggestion:

`failure(BiometricError.given(error as NSError) ?? error)`

Reply:
Our intention with `BiometricError` enum is to only catch the errors related to biometric authentication, the only issue is that they are spread into two other OS enums (LAError & OSStatus Errors) and those enums carry a lot of errors that cover too many other things, so if we catch that it is an error that we care for the biometric authentication we return a BiometricError, and in the other case we return the real error, so it is not ignored either way.

Thats why that `if & else` is needed.